docs/reference/analysis/tokenfilters/synonym-tokenfilter.asciidoc

```diff
@@ -107,6 +107,13 @@ set to `false` no mapping would get added as when `expand=false` the target mapp
 `expand=true` then the mappings added would be equivalent to `foo, baz => foo, baz` i.e, all mappings other than the
 stop word.
 
+[WARNING]
```

Comment:
Sorry to keep nagging about this - I think this should go under the `synonyms_set` config (line 35).

Reply:
I put it here because it's next to the `PUT index` example, and that's the action we're warning about. But I'll defer to you and move it. 👍